test/e2e/garbage_collector.go

```diff
@@ -275,4 +333,50 @@ var _ = framework.KubeDescribe("Garbage collector", func() {
 		}
 		gatherMetrics(f)
 	})
+
+	It("[Feature:GarbageCollector] should delete RS created by deployment when not orphaning", func() {
```

Comment:
Could you remove the `[Feature:GarbageCollector]` from all these tests? I think the submit-queue e2e job skips the test if it has [Feature: XXX] in the name.